src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml

```diff
@@ -1,4 +1,8 @@
 @{
+    if (Model.Type == null)
+    {
+        return;
+    }
```

Comment:
This is not a bootstrap related change. Propose this in a different Pull Request..

Reply:
done.